pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-standalone/src/main/java/org/apache/pinot/plugin/ingestion/batch/standalone/SegmentGenerationJobRunner.java

```diff
@@ -160,6 +162,9 @@ public void init(SegmentGenerationJobSpec spec) {
     LOGGER.info("Creating an executor service with {} threads(Job parallelism: {}, available cores: {}.)", numThreads,
         jobParallelism, Runtime.getRuntime().availableProcessors());
     _executorService = Executors.newFixedThreadPool(numThreads);
+
+    // Set up for recording multiple failures while building segments.
```

Comment:
(minor) The comment is a little bit confusing. Suggest updating it to reflect that we record the first failure

Reply:
Good catch, thanks. Fixed.